src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -1,21 +1,31 @@
 (ns status-im.ui.screens.communities.community-overview-redesign
-  (:require  [status-im.i18n.i18n :as i18n]
-             [quo.react-native :as rn]
-             [quo2.components.markdown.text :as text]
-             [quo2.components.buttons.button :as button]
-             [quo2.components.list-items.preview-list :as preview-list]
-             [quo2.components.list-items.channel :as channel]
-             [quo2.components.dividers.divider-label :as divider-label]
-             [quo2.components.community.community-view :as community-view]
-             [quo2.components.tags.status-tags :as status-tags]
-             [status-im.ui.screens.communities.request-to-join-bottom-sheet-redesign :as request-to-join]
-             [status-im.ui.screens.communities.community-options-bottom-sheet :as options-menu]
-             [status-im.ui.components.list.views :as list]
-             [status-im.utils.handlers :refer [<sub >evt]]
-             [status-im.ui.screens.communities.styles :as styles]
-             [quo2.foundations.colors :as colors]
-             [quo2.components.navigation.page-nav :as page-nav]
-             [status-im.ui.screens.communities.icon :as communities.icon]))
+  (:require [quo.react-native :as rn]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.community.community-view :as community-view]
+            [quo2.components.community.token-gating :as token-gating]
+            [quo2.components.dividers.divider-label :as divider-label]
+            [quo2.components.list-items.channel :as channel]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.navigation.page-nav :as page-nav]
+            [quo2.components.tags.status-tags :as status-tags]
+            [quo2.foundations.colors :as colors]
+            [status-im.constants :as constants]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.react-native.resources :as resources]
+            [status-im.ui.components.list.views :as list]
+            [status-im.ui.screens.communities.community-options-bottom-sheet :as options-menu]
+            [status-im.ui.screens.communities.icon :as communities.icon]
+            [status-im.ui.screens.communities.request-to-join-bottom-sheet-redesign :as request-to-join]
+            [status-im.ui.screens.communities.styles :as styles]
+            [status-im.utils.handlers :refer [<sub >evt]]
+            [status-im.utils.utils :as utils]))
+
+(def knc-token-img (js/require "../resources/images/tokens/mainnet/KNC.png"))
+(def mana-token-img (js/require "../resources/images/tokens/mainnet/MANA.png"))
+(def rare-token-img (js/require "../resources/images/tokens/mainnet/RARE.png"))
+(def eth-token-img (js/require "../resources/images/tokens/mainnet/ETH.png"))
+(def dai-token-img (js/require "../resources/images/tokens/mainnet/DAI.png"))
```

Comment:
Kindly fix the require list in the file, I think Calva automatically sorts requires and therefore making unnecessary changes to files